From: Diane Trout <diane@ghic.org>
Date: Mon, 24 Apr 2017 16:38:34 -0700
Subject: Remove github username and project id

Including it triggers a privacy breach in the locally generated
html files.

--- a/docs/conf.py
+++ b/docs/conf.py
@@ -89,9 +89,6 @@
     'logo_text_align': 'center',
     'show_powered_by': False,
     'show_related': True,
-    'github_user': 'grantjenks',
-    'github_repo': 'python-sortedcollections',
-    'github_type': 'star',
 }
 
 # Add any paths that contain custom static files (such as style sheets) here,
